src/main/java/nextstep/users/domain/NextStepUsers.java

```diff
@@ -0,0 +1,32 @@
+package nextstep.users.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class NextStepUsers {
+
+  private static final String MAX_ENROLLMENT_MESSAGE = "해당 세션의 수강 인원이 만석되었습니다.";
+  private final int maxUserEnrollment;
+  private final List<NextStepUser> nextStepUsers;
+
+  public NextStepUsers(int maxUserEnrollment) {
+    this.nextStepUsers = new ArrayList<>();
+    this.maxUserEnrollment = maxUserEnrollment;
+  }
+
+  public void enroll(NextStepUser nextStepUser) {
+    canEnrollUser();
```

Comment:
```suggestion
    if (!canEnrollUser()){
      throw new IllegalArgumentException(MAX_ENROLLMENT_MESSAGE);
    }
```

can, has, is 와 같은 네이밍은 boolean 리턴을 받게 메서드를 설계합니당 :)